package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "7.0.16",
+  "version": "7.0.16a",
```

Comment:
Version 7.0.16 is what is already in Test and pending deployment to Prod. This is the build I am patching. I cannot change this to 7.0.17 because we already have that version in main branch.

Note: At the top of this page, you can see I want to merge this into a release branch, ie

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/b4b242dc-c1c2-4a2b-9be7-17dc1b57a2f3)

Note: After merging this PR, I will manually deploy it to Dev, and then manually deploy it to Test. (Only main branch is automatically deployed to Dev.)